sullivan_a06c.js

```diff
@@ -0,0 +1,59 @@
+/*  Anna Sullivan
+    sullivan_a06c.js
+    INFO2124WW
+    Thoendel
+    1/25/2020
+*/
+//winning numbers: 22, 42, 43, 47, 61;
+
+let lottoPicks = [];
+let powerBallPick = getLottoNumber(); 
+let winnings = 0; 
+
+/* WRITE YOUR SOLUTION BETWEEN THIS LINE */
+
+/*do.. while to push the return value of getLottoNumber into lottoPicks array when less than 5*/
+do {
+lottoPicks.push(getLottoNumber());
+} while (lottoPicks < 5);
+
+for(let lottoPick of lottoPicks){
+/*for loop iterates through length of lottoPicks array*/
+    switch(lottoPick){ /*switch to determine if an element of lottoPick in lottoPicks array matches the winnings numbers then adds 10*/
+        case 22:
+                winnings + 10;

```

Comment:
in these cases - you should output the matched number as per the sample output: "Matched 22!"
Also - you will never win the way this is written - you need to add 10 to the winnings value and store it in winnings: winnings += 10;
-6